client/src/types.ts

```diff
@@ -86,6 +86,18 @@ export interface Lesson {
   deleted?: boolean;
 }
 
+export interface Media {
+  link: string;
+  type: string;
+  props: Array<{ name: string; value: string }>;
+}
+
+export enum MediaType {
+  NONE = "none",
```

Comment:
in all cases, let's prefer to use `""` as the value that indicates `NONE` unless there's a strong reason to do otherwise